packages/dds/tree/src/feature-libraries/forest-summary/incrementalSummaryBuilder.ts

```diff
@@ -61,6 +61,23 @@ export const ForestSummaryTrackingState = {
 export type ForestSummaryTrackingState =
 	(typeof ForestSummaryTrackingState)[keyof typeof ForestSummaryTrackingState];
 
+/**
+ * The properties of a chunk tracked during the loading process.
+ * These are used to identify a chunk when it is decoded and recreate the tracking state
+ * as it was when the summary that the client is loading from was generated.
```

Comment:
I think it would be better to document this more in terms of what its semantics are rather than how it's used.

That way it's possible to know code using it is doing so correctly, even if that code is not exactly the code we have now.

Maybe something like:
```suggestion
 * An encoded chunk, paired with a location it can be reused / reloaded from.
 * @remarks
 * This identifies a location in a specific summary where `content` was loaded from.
 * 
 * When summarizing, Fluid always ensures the summary that the summary client is allowed to reuse content from is the one it loaded from,
 * so tracking this on load is sufficient for now:
 * there is no need to track the equivalent data when summarizing.
```